animation_nodes/nodes/mesh/add_mesh_vertex_color.py

```diff
@@ -0,0 +1,69 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... base_types import AnimationNode, VectorizedSocket
+from ... data_structures import Color, ColorList, VirtualColorList
+from .. color.c_utils import getLoopColorsFromVertexColors, getLoopColorsFromPolygonColors
+
+colorModeItems = [
+    ("LOOP", "Loop", "Get color of every loop vertex", "NONE", 0),
+    ("VERTEX", "Vertex", "Get color of every vertex", "NONE", 1),
+    ("POLYGON", "Polygon", "Get color of every polygon", "NONE", 2)    
+]
+
+class AddMeshVertexColorNode(bpy.types.Node, AnimationNode):
```

Comment:
Perhaps we should change the name to `InsertVertexColorLayerNode`, to be more uniform and to make it easier to search for. Do you agree?

Reply:
Omar, `Add Mesh Vertex Color` and  `Get Mesh Vertex Color` nodes do similar jobs, i.e., one node insert the vertex color layer and its colors, and the other gets the colors of a vertex color layer. I think if their names are similar then it would be nice. 

Add Mesh Vertex Color -> Insert Vertex Color Layer
Get Mesh Vertex Color -> Get Vertex Color Layer 

or

Add Mesh Vertex Color -> Mesh Vertex Color Output  or  Vertex Color Layer Output
Get Mesh Vertex Color -> Mesh Vertex Color Input  or  Vertex Color Layer Input

Reply:
Yes, sure. This looks good:

Add Mesh Vertex Color -> Insert Vertex Color Layer
Get Mesh Vertex Color -> Get Vertex Color Layer

Reply:
Okay.